PluginCore/DockPanelSuite/Docking/DockPanel.FocusManager.cs

```diff
@@ -416,13 +416,13 @@ internal void SetActiveContent()
                     return;
 
                 if (m_activeContent != null)
-                    m_activeContent.DockHandler.IsActivated = false;
+                    m_activeContent.DockHandler.SetIsActivated(false);
```

Comment:
Just checked.
It seems they still have it the way it was 
https://github.com/dockpanelsuite/dockpanelsuite/blob/master/WinFormsUI/Docking/DockPanel.FocusManager.cs#L468
https://github.com/dockpanelsuite/dockpanelsuite/blob/master/WinFormsUI/Docking/DockContentHandler.cs#L706